main.go

```diff
@@ -697,11 +697,33 @@ func DeleteBackupAction(c *cli.Context) error {
 	if len(folder) != 0 {
 		name = fmt.Sprintf("%s/%s", folder, name)
 	}
-	for _, p := range []string{name, fmt.Sprintf("%s.%s", name, compressedExtension)} {
```

Comment:
if you have a backup: `backups/backup1.zip`, this logic fails because it tries to delete the uncompressed backup first (`backups/backup1`), which will not exist and thus triggers an early return.